qa/L0_shared_memory/shared_memory_test.py

```diff
@@ -323,6 +324,29 @@ def test_unregisterall(self):
             self.assertTrue(len(status_after.regions) == 0)
         self._cleanup_server(shm_handles)
 
+    def test_infer_offset_out_of_bound(self):
+        # Shared memory offset outside output region - Throws error
```

Comment:
Is it possible to either loop around or go backwards in terms of offset? i.e. can/should we test a target address before the region rather than after it?

Reply:
> Is it possible to either loop around or go backwards in terms of offset?

Yes. We are dealing with unsigned values at the core, so `18446744073709551584` offset is actually looping around, which ends up 32 bytes before the start of the original shared memory address.